pycbc/workflow/core.py

```diff
@@ -198,7 +198,10 @@ def __init__(self, cp, name,
             raise TypeError("Failed to find %s executable " 
                             "at %s" % (name, exe_path))
         
-        self.add_pfn(exe_path)
+        if exe_path.startswith('gsiftp://'):
+            self.add_pfn(exe_path,site='pycbc-code')
```

Comment:
i think it's better to use something more specific here so that it can have an entry in the site catalog. "nonlocal" is a bit too generic.